tests/recipe/CREATE_recipes.test.js

```diff
@@ -0,0 +1,130 @@
+const supertest = require("supertest");
+const app = require("../../app");
+const Recipe = require("../../models/recipe");
+const User = require("../../models/user");
+const api = supertest(app);
+const mongoose = require("mongoose");
+
+const {
+  initialUsers,
+  recipesInDb,
+  recipeToAdd,
+  recipeReview,
+} = require("../test_helper");
+const STATUS_CODE = require("../../shared/errorCode");
+
+let authHeader;
+let userId;
+let recipeId;
+let token;
+
+describe("recipe api", () => {
+  beforeEach(async () => {
+    await User.deleteMany({});
+    const newUser = initialUsers[1];
+    let response = await api.post("/api/users").send(newUser);
```

Comment:
If you don't need to check the response, then you don't need as a local variable.
```suggestion
    await api.post("/api/users").send(newUser);
```